packages/components/src/internal/components/base/CreatedModified.tsx

```diff
@@ -141,7 +141,10 @@ export class CreatedModified extends Component<CreatedModifiedProps, State> {
             const displayTxt = serverDate ? moment(timestamp).from(serverDate) : moment(timestamp).fromNow();
 
             return (
-                <span title={this.formatTitle(config)} className={classNames('createdmodified', className)}>
+                <span
+                    title={this.formatTitle(config)}
+                    className={classNames('createdmodified', 'gray-text', className)}
```

Comment:
Aren't you removing the `createdmodified` class?

Reply:
I removed the styles for it, but the classname is still used for a selenium test locator so I left it in for that.